youtube_transcript_api/test/test_api.py

```diff
@@ -53,11 +53,11 @@ def test_get_transcript__correct_language_is_used(self):
     def test_get_transcript__fallback_language_is_used(self):
         httpretty.register_uri(
             httpretty.GET,
-            'https://www.youtube.com/api/timedtext',
-            body=''
+            'https://www.youtube.com/watch',
+            body=load_asset('youtubeWW1.html.static')
```

Comment:
What is the difference between `youtubeWW1.html.static` and  `youtube.html.static`? Could this be named more descriptively?

Reply:
`youtubeWW1.html.static` is the html from the video mentioned in issue #35. I figured this would show that the bug was fixed. In addition, there was some problem with the original test not passing with the changes I made; its been a while so I can't  remember why exactly it was failing off the top.

I could rename it to something like `youtube_nl_en_captions.html.static` maybe? I'm not exactly sure what is the best way to differentiate the youtube static html files.